modules/Bio/EnsEMBL/PipeConfig/BuildRegulationMart_conf.pm

```diff
@@ -271,70 +216,7 @@ sub pipeline_analyses {
                        'max_dropdown' => $self->o('max_dropdown'),
                        'base_name' => 'external_feature' },
       -analysis_capacity => 1,
-      -flow_into => ['generate_meta_peaks'],
```

Comment:
I think we still need the generate meta peaks, because this dataset is still in use for regulation mart (I re-added it after having built it for 104)

Reply:
Okay, I guess `AddExtraMartIndexesPeaks` analysis should be reverted back as well, correct?

Reply:
Yes, well spotted.